packages/components/calendar/src/use-calendar.ts

```diff
@@ -18,13 +18,15 @@ interface Props extends UseCalendarBaseProps {
    * Props for the button picker, which is used to select the month, year and expand the header.
    */
   buttonPickerProps?: ButtonProps;
+  renderCellContent?: (date: CalendarDate) => React.ReactNode;
```

Comment:
remove this line and retrieve `renderCellContent` from `useCalendarBase`

Reply:
```
       <Calendar {...args} calendarWidth={340}>
          {(date) => (
            <CalendarCellContent>
              <CalendarCellHeader />
              <CalendarCellBody>
                <div className="flex flex-col w-full text-tiny gap-0.5 px-0.5">
                   // Cell
                </div>
              </CalendarCellBody>
            </CalendarCellContent>
          )}
        </Calendar>
```

Changed renderCellContent to children.